packages/grafana-ui/src/components/TableInputCSV/TableInputCSV.tsx

```diff
@@ -78,10 +81,10 @@ export class UnThemedTableInputCSV extends React.PureComponent<Props, State> {
           placeholder="Enter CSV here..."
           value={this.state.text}
           onChange={this.onTextChange}
-          className="gf-form-input"
+          className={classNames(styles.textarea, 'gf-form-input')}
```

Comment:
@thisisobate shouldn't you use `cx` from emoition to combine class names instead? Maybe I'm missing something, WDYT @jackw

Reply:
You're right Hugo.
Would update the changes right away